installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc

```diff
@@ -8,10 +8,18 @@ toc::[]
 
 After successfully deploying an installer-provisioned cluster, consider the following postinstallation procedures.
 
+// Configuring NTP for disconnected clusters
 include::modules/ipi-install-configuring-ntp-for-disconnected-clusters.adoc[leveloffset=+1]
 
+// Enabling a provisioning network after installation
 include::modules/nw-enabling-a-provisioning-network-after-installation.adoc[leveloffset=+1]
 
+// Configuring a user-managed load balancer
```

Comment:
This comment should be on line 20 and line 20 comment should be here

Reply:
Good catch, thanks!